src/features/vote/styled/ActiveRequests.styled.tsx

```diff
@@ -43,8 +42,13 @@ export const Wrapper = styled.div`
   }
 `;
 
+export const DescriptionWrapper = styled.div`
+  max-width: 550px;
+`;
 export const Description = styled.span`
   color: #818180;
+  line-height: 1.57;
+  font-size: 14px;
```

Comment:
We probably want all font-sizes to be in REM as they would scale when the users zooms or changes their default font size. 
One trick you can use to specify the font size in `px` while setting it in `rem` is doing `font-size: ${14 / 16}rem`